packages/ibm-products/src/components/GetStartedCard/GetStartedCard.mdx

```diff
@@ -5,6 +5,8 @@ import * as stories from './GetStartedCard.stories';
 
 # GetStartedCard
 
+[Onboarding overview](https://pages.github.ibm.com/carbon/ibm-products/components/onboarding/overview/)
```

Comment:
```suggestion
[Get started pattern](https://pages.github.ibm.com/carbon/ibm-products/patterns/onboarding-flows/orientation/#get-started)
```

“Usage guidelines” or maybe just “pattern”?

Reply:
These are not the same. See https://github.com/carbon-design-system/ibm-products/pull/6989#discussion_r1973663274

Reply:
Hah, okay. I was just about to ask about the StoryDocsPage as I was rereviewing. :)